### PR TITLE
Readme sync

### PR DESCRIPTION
**Pull request by ironmaniiith**
Refer #211

### CODE REVIEW

**Comment by jdalton** [comment]
💃

**Comment by ironmaniiith** [comment]
Updating README for `is.iphone`, `is.ipad` and `is.ipod` is remaining.

File: README.md [hunk 1]
```diff
@@ -1638,11 +1684,21 @@ is.opera();
...
+
+is.opera('>=35');
+=> true if current version of opera is greater than or equal to 35
+
+is.not.opera('<20');
+=> true if current version of opera is not less than 20
 ```
 
-is.safari()
+is.safari(range:string)
```
Comment (jdalton) [inline README.md]: Maybe as `range:number|string` ?

Comment (ironmaniiith) [inline README.md]: I was having the same opinion, will do it. :+1:

**Comment by ironmaniiith** [comment]
@jdalton , I tested for all the browsers except iphone, ipad and ipod, can you please suggest me some resources where I can verify these methods.

**Comment by jdalton** [comment]
>  I tested for all the browsers except iphone, ipad and ipod, can you please suggest me some resources where I can verify these methods.

Emulators on OSX or a trial of [BrowserStack](https://www.browserstack.com/).

**Comment by ironmaniiith** [comment]
Thanks, already used BrowserStack, but was facing some problems with devtools, anyways managed a way out.

**Pull request merged**